twelvelabs/base_client.py

```diff
@@ -37,10 +37,18 @@ def _request(self, method: str, url: str, **kwargs) -> Any:
         except httpx.HTTPStatusError as e:
             raise self._make_status_error(e.response)
 
-        if len(response.content) > 0 and "application/json" in response.headers.get("Content-Type", ""):
+        if "chunked" in response.headers.get("Transfer-Encoding", ""):
+            return response
```

Comment:
out of curiosity, when would it be the case where "chunked" is included in this header for the regular generate api call?

Reply:
Oh, previously I tried to handle this with a regular request method, but decided to leverage `stream` with HTTPX. This code was leftover from before, so it has been removed! Thanks